src/app/components/AmpGeo/index.jsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+
+const configuration = {
+  AmpBind: true,
+  ISOCountryGroups: {
+    eea: [
+      'at',
+      'ax',
+      'be',
+      'bg',
+      'bl',
+      'cy',
+      'cz',
+      'de',
+      'dk',
+      'ea',
+      'ee',
+      'es',
+      'fi',
+      'fr',
+      'gb',
+      'gf',
+      'gg',
+      'gi',
+      'gp',
+      'gr',
+      'hr',
+      'hu',
+      'ic',
+      'ie',
+      'im',
+      'is',
+      'it',
+      'je',
+      'li',
+      'lt',
+      'lu',
+      'lv',
+      'mf',
+      'mq',
+      'mt',
+      'nc',
+      'nl',
+      'no',
+      'pf',
+      'pl',
+      'pm',
+      'pt',
+      're',
+      'ro',
+      'se',
+      'si',
+      'sj',
+      'sk',
+      'tf',
+      'va',
+      'wf',
+      'yt',
+    ],
+    gbOrUnknown: ['gb', 'gg', 'im', 'je', 'uk', 'unknown'],
+  },
+};
+
+export const AMP_GEO_SCRIPT = (
+  <script
+    async
+    custom-element="amp-geo"
+    src="https://cdn.ampproject.org/v0/amp-geo-0.1.js"
+  />
+);
+
+const AmpGeo = () => (
```

Comment:
Can we make sure a PR is raised to deprecate amp-geo in psammead?

Reply:
Yeah I can do that 👍